stores/sql.go

```diff
@@ -31,14 +32,20 @@ type (
 		LongTxDuration                time.Duration
 	}
 
+	Explorer interface {
+		Enabled() bool
+	}
+
 	// SQLStore is a helper type for interacting with a SQL-based backend.
 	SQLStore struct {
 		alerts    alerts.Alerter
 		db        sql.Database
 		dbMetrics sql.MetricsDatabase
 		logger    *zap.SugaredLogger
 
-		walletAddress types.Address
+		explorerDisabled bool
```

Comment:
Not too sure how we can avoid these two state variables on the store if we want to have the correct behaviour in the setting store. I chose to put it on the store (and not in the migrations) because there we want to return defaults if whatever is persisted is not making sense.

Reply:
The setting store doesn't care right? It just returns either the pinned settings that it has stored or if none exist, returns the default settings. Reading settings shouldn't be affected by the explorer.